src/styles/main.css

```diff
@@ -31,6 +31,7 @@ body {
   font-size: 13.5px;
 }
 #footer {
+  float: right;
```

Comment:
Is there reason why the footer is floating right? I wasn't able to see any difference in the appearance of the footer when this css was removed and when it was present.

Reply:
It has to do with when the browser window is made narrower, otherwise it bumps to the right side of the page.

Reply:
@jermnelson see comment above, but it does looks like there is some further CSS refinement that needs to happen. Not sure if this is the PR for that given that we will be doing work on the form later on.

Reply:
hmmm @jgreben i'm not seeing the problem when the browser is made narrower. Could you show me whenever you have time?

Reply:
^saw when webpage was enbiggened or however you spell it

Reply:
👌